### PR TITLE
[WNMDGS-2975] Change focus color from orchid to copper on CMSgov theme

### PR DESCRIPTION
**Pull request by malloryiden**
## Summary
Follow up PR related to [this one](https://github.com/CMSgov/design-system/pull/3227).

Isolated only the focus color change for CMSgov.

## How to test

1. Confirm that CMSgov's dark focus color changed from orchid/500 to copper/500.
2. Confirm that the copper focus state is being used on the docsite.

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.
- [x] Selected appropriate release milestone

### CODE REVIEW

**Comment by malloryiden** [comment]
Why does the build keep failing? 😩 May need to pair with an eng to sort this out.

File: packages/design-system-tokens/src/tokens/System.Value.json [hunk 1]
```diff
@@ -2663,7 +2663,7 @@
       "$extensions": {
         "com.figma": {
           "hiddenFromPublishing": false,
-          "scopes": ["GAP", "WIDTH_HEIGHT", "TEXT_CONTENT"],
+          "scopes": ["GAP", "TEXT_CONTENT", "WIDTH_HEIGHT"],
```
Comment (jack-ryan-nava-pbc) [inline packages/design-system-tokens/src/tokens/System.Value.json:2666]: Blerg. These changes will keep happening until v12 is released. See PR #3226 
I guess I can assign to a new milestone.

**Comment by tamara-corbalt** [comment]
Hey @malloryiden, everything looks good with the changes. But, when I tested the doc site locally, I still saw the violet outline in the focus state under the cmsgov theme.

Here are the steps I followed in the root of our repo:
```
yarn clean
yarn install
yarn build
yarn build:docs
```
After running `yarn build:docs` I noticed that the file packages/docs/static/themes/cmsgov-theme.css was modified, specifically removing the line --color-focus-dark: #bd13b8;. After this change, I then saw the copper outline in the focus state.

**Comment by jack-ryan-nava-pbc** [comment]
@malloryiden Coupla things:
1. The focus change doesn't show up on the Doc Site unless you run `yarn build:docs`, this will change the cmsgov-theme.css file. Can you run that and commit the changes?
2. You might need to rerun some snapshot tests as a result
3. Checking the contrast between the hover color and various backgrounds: it's ok to use for UI elements and focus.

### Focus on white:
Screenshot 2024-09-25 at 9 48 30 AM

### Focus on blue:
Screenshot 2024-09-25 at 9 49 05 AM

File: packages/design-system-tokens/src/tokens/System.Value.json (hunk 1 above)
Comment (malloryiden) [reply packages/design-system-tokens/src/tokens/System.Value.json:2666]: I was wondering where those came from!

**Review by jack-ryan-nava-pbc — APPROVED** [review]
Approve!

**Review by tamara-corbalt — APPROVED** [review]
Looks good to me!

**Pull request merged**